desafio-02/guiribeirodev/javascript/desafio-02.js

```diff
@@ -0,0 +1,29 @@
+//Função "listaPrimo" é responsável por criar a lista dos números primos do número inicial até o numero final.
+function listaPrimo(numeroInicio, numeroFim) {
+  for (numeroInicio = 1; numeroInicio <= numeroFim; numeroInicio++) {
+    verificaPrimo(numeroInicio, numeroFim)
+  }
+}
+
+/* Função "verificaPrimo()"" irá verificar se o número é primo, a verificação é feita 
+com base na premissa que todo número primo irá ter apenas dois divisores, 
+o número 1 e ele próprio.*/
+function verificaPrimo(numeroInicio) {
+  let divisor = 0
+
+  for (let count = 1; count <= numeroInicio / 2; count++) {
+    if (numeroInicio % count == 0) {
+      divisor++
+    }
+
+    if (divisor > 1) {
+      return divisor
```

Comment:
Como não está utilizando o resultado deste método, não seria legal colocar apenas `return`?

Reply:
verdade, não tinha reparado nisso, porém analisando o código, no lugar do return vazio pensei em substituir por um break, visto que não retorna nada, o que você acha ser melhor?

Reply:
Como a nível de código o impacto é o mesmo, sugiro colocar o que acha mais fácil de entender se for outra pessoa lendo o código(para mim não faz tanta diferença)

Reply:
feito, alterei por break, em minha opinião é mais declarativo, obrigado!